scripts/OMakefile

```diff
@@ -46,7 +46,7 @@ install:
 	mkdir -p $(DESTDIR)/etc/rc.d/init.d
 	$(IPROG) init.d-xapi $(DESTDIR)$(LIBEXECDIR)/xapi-init
 	$(IPROG) init.d-xapissl $(DESTDIR)$(LIBEXECDIR)/xapissl
-	$(IPROG) init.d-management-interface $(DESTDIR)/etc/rc.d/init.d/management-interface
+	$(IPROG) init.d-management-interface $(DESTDIR)$(LIBEXECDIR)/legacy-management-interface
```

Comment:
Do we still use this script, and if so do we need to add the systemd files for it?